### PR TITLE
Update org-data-overview.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/Viva/issues/520

### CODE REVIEW

File: Viva/insights/advanced/admin/org-data-overview.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Advanced insights automatically collects collaboration data from Microsoft 365.
...
+The advanced insights app can get organizational data in one of two ways: through Azure Active Directory, which is the default setting, or through an organizational data file that you as an admin upload. Azure Active Directory automatically syncs with the advanced insights app and provides data for the **PersonId**, **ManagerId**, **Organization**, **Domain** and **Timezone** attributes. Domain will be based on the email address value of the user in Azure Active Directory and time zone is synced from the Outlook settings. If you want to include more attributes, for example, those we discuss in [Prepare organizational data](prepare-org-data.md#attribute-reference), then you'd want to choose a data file as your source.
```
Comment (JohanFreelancer9) [inline Viva/insights/advanced/admin/org-data-overview.md:30]:
```suggestion
The advanced insights app can get organizational data in one of two ways: through Azure Active Directory, which is the default setting, or through an organizational data file that you as an admin upload. Azure Active Directory automatically syncs with the advanced insights app and provides data for the **PersonId**, **ManagerId**, **Organization**, **Domain**, and **Timezone** attributes. Domain will be based on the email address value of the user in Azure Active Directory and Timezone is synced from the Outlook settings. If you want to include more attributes, for example, those we discuss in [Prepare organizational data](prepare-org-data.md#attribute-reference), then you'd want to choose a data file as your source.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@dariomws

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@lilyolason 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dariomws** [comment]
@lilyolason
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by vstanche** [comment]
Sorry don't understand why the request was closed, do not see anything changed in the public article?

Can you please advise?

**Comment by dariomws** [comment]
@vstanche, you'll be able to see the updates in 24 hours.

**Comment by vstanche** [comment]
Hello Dario,

Please be aware that updates are not there yet

***@***.***

Is that normal? Or the updates were done on another place?

Kind Regards,
***@***.***
***@***.***
Veselin Stanchev

Employee Experience / Viva SME, FastTrack for M365 EMEA

Customer Service & Support
***@***.******@***.***>





Microsoft Edge
The browser for business | Standardize on one browser with Microsoft security and innovation
Get assistance from FastTrack<https://www.microsoft.com/fasttrack> | Learn about Edge<https://www.microsoft.com/edge>
This and subsequent emails on the same topic are for discussion and information purposes only. Only those matters set out in a fully executed agreement are legally binding. This email may contain confidential information and should not be shared with any third party without the prior written agreement of Microsoft. If you are not the intended recipient, take no action and contact the sender immediately.

Microsoft Limited (company number 01624297) is a company registered in England and Wales whose registered office is at Microsoft Campus, Thames Valley Park, Reading RG6 1WG
If you have any feedback about my work, please let either myself or my manager know -  Mark Bebbington ***@***.******@***.***>

**Comment by lilyolason** [comment]
@dariomws and @vstanche , I'm checking in on this.

**Comment by vstanche** [comment]
Thanks very much and really appreciate adding this content.

Just want to clarify something. According to the page

Azure Active Directory automatically syncs with the advanced insights app and provides data for these attributes:
*        PersonId
*        ManagerId
*        Organization
In your imported data, you'll also notice:
*        Domain, which Viva Insights gets from users' primary SMTP addresses.
*        TimeZone, which Viva Insights gets from users' Outlook/Exchange settings.

I think this is likely to create some confusion for our customers, why are the 1st three attributes categorized differently in comparison to domain and timezone. Domain and timezone are not automatically synced but they will be also noticed?

I am trying to determine if there is a difference in the mechanism between the 1st three and the other two?


Kind Regards,
***@***.***
***@***.***
Veselin Stanchev

Employee Experience / Viva SME, FastTrack for M365 EMEA

Customer Service & Support
***@***.******@***.***>





Microsoft Edge
The browser for business | Standardize on one browser with Microsoft security and innovation
Get assistance from FastTrack<https://www.microsoft.com/fasttrack> | Learn about Edge<https://www.microsoft.com/edge>
This and subsequent emails on the same topic are for discussion and information purposes only. Only those matters set out in a fully executed agreement are legally binding. This email may contain confidential information and should not be shared with any third party without the prior written agreement of Microsoft. If you are not the intended recipient, take no action and contact the sender immediately.

Microsoft Limited (company number 01624297) is a company registered in England and Wales whose registered office is at Microsoft Campus, Thames Valley Park, Reading RG6 1WG
If you have any feedback about my work, please let either myself or my manager know -  Mark Bebbington ***@***.******@***.***>

**Pull request merged**